cds/modules/fixtures/cli.py

```diff
@@ -247,6 +247,11 @@ def page_data(page):
              description='FAQ',
              content=page_data('faq.html'),
              template_name='invenio_pages/dynamic.html'),
+        Page(url='/guide/search',
+             title='Search guide',
+             description='Search guide',
+             content=page_data('search.html'),
```

Comment:
Actually, tests are not passing because this path is wrong. I see 2 options:

- move the search.html in `data/pages/search.html` (remove `guides` folder), but keep the url to `/guide/search`
OR
- change that line to `page_data('guides/search.html')`

Reply:
Done, second option ("guide/..." not "guides/...")